features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Listing.jsx

```diff
@@ -0,0 +1,192 @@
+                        />
+                    </Typography>
+                    <div align='center'>
+                        <Button className={classes.buttonStyle} variant='outlined'>
+                            <Typography className={classes.heading} variant='h6'>
+                                <FormattedMessage
+                                    id='ServiceCatalog.Listing.Listing.get.started'
+                                    defaultMessage='Get Started'
+                                />
+                            </Typography>
+                        </Button>
+                    </div>
+                </Grid>
+                <Grid item md={4}>
+                    <div align='center'>
+                        <img
+                            className={classes.preview}
+                            src={Configurations.app.context + '/site/public/images/wso2-intg-service-sample-icon.svg'}
+                            alt='Add Sample Service'
+                        />
+                    </div>
+                    <Typography className={classes.heading} variant='h4' align='center'>
+                        <FormattedMessage
+                            id='ServiceCatalog.Listing.Listing.Heading2'
+                            defaultMessage='Add a sample'
+                        />
+                    </Typography>
+                    <Typography align='center'>
+                        <FormattedMessage
+                            id='ServiceCatalog.Listing.Listing.Heading2.subHeading'
+                            defaultMessage='Integration Service'
+                        />
+                    </Typography>
+                    <Typography align='center' className={classes.spacing}>
+                        <FormattedMessage
+                            id='ServiceCatalog.Listing.Listing.description2'
+                            defaultMessage={'From creating and publishing an API to securing, rate-limiting, addresses'
+                            + ' all aspects of API Management.'}
+                        />
+                    </Typography>
+                    <div align='center'>
+                        <Button className={classes.buttonStyle} variant='outlined'>
+                            <Typography className={classes.heading} variant='h6'>
+                                <FormattedMessage
+                                    id='ServiceCatalog.Listing.Listing.add.sample.service'
+                                    defaultMessage='Add Sample Service'
+                                />
+                            </Typography>
+                        </Button>
+                    </div>
+                </Grid>
+                <Grid item md={2} />
+            </Grid>
+        </div>
+    );
+}
+
+Listing.propTypes = {
+    classes: PropTypes.shape({}).isRequired,
+};
+
+export default injectIntl(withStyles(styles, { withTheme: true })(Listing));
```

Comment:
Shall we use hooks replacement for Intl & styles HOC components used in here
I:e

```
useStyles()
```

Reply:
Fixed with https://github.com/tharikaGitHub/carbon-apimgt/commit/e465cc6f9929d6ab34f6d75e967fd5ef825445f2